scst-store/tls-configuration.hbs.md

```diff
@@ -0,0 +1,53 @@
+
+# Setting up TLS in Metadata Store
+
+###  <a id="Setting up tls namespace and secretName"></a>Setting up tls namespace and secretName
+
+In `tap-values` file you can set namespace and secretName as shown below.
+
+---
+```yaml
+tls:
+  namespace: "The targeted namespace for secret consumption by the HTTPProxy."
+  secretName: "The name of secret for consumption by the HTTPProxy."
+```
+---
+In `tap-values` file you also set Minimum TLS version supported by metadata store and ciphers supported.
+
+---
+```yaml
+tls.server.minTLSVersion:
+  type: string
+  description: 'Minimum TLS version supported. Value must match version names from https://golang.org/pkg/crypto/tls/#pkg-constants. (default "VersionTLS12")'
+  default: "VersionTLS12"
+tls.server.rfcCiphers:
+  type: array
+  description: 'List of cipher suites for the server. Values are from tls package constants (https://golang.org/pkg/crypto/tls/#pkg-constants). If omitted, the default Go cipher suites will be used'
+  default: 
+    - TLS_AES_128_GCM_SHA256
+    - TLS_AES_256_GCM_SHA384
+    - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+    - TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+    - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+    - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+```
+---
+### Complete TLS setting might look like as below.
+
+---
+```yaml
+tls:
+  namespace: ""
+  secretName: ""
+  server:
+    minTLSVersion: VersionTLS12
+    rfcCiphers:
+      - TLS_AES_128_GCM_SHA256
+      - TLS_AES_256_GCM_SHA384
+      - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+      - TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+      - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+      - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+```
+---
+#### <span style="color:red">Note: Metadata store doesn't support TLS1.0, TLS1.1 and TLS3.0. TLS 1.3 support is planned in future releases</span>.
```

Comment:
it should be TLS1.3 not TLS3.0

Reply:
Should we even document `minTLSVersion`? because the user is not able to use other tls settings

Reply:
Very good point. Let's discuss whenever you have time.